test_src/sljitTest.c

```diff
@@ -10743,7 +10751,16 @@ int sljit_test(int argc, char* argv[])
 		printf("all tests are " COLOR_GREEN "PASSED" COLOR_DEFAULT " ");
 	else
 		printf(COLOR_RED "%d" COLOR_DEFAULT " (" COLOR_RED "%d%%" COLOR_DEFAULT ") tests are " COLOR_RED "FAILED" COLOR_DEFAULT " ", TEST_COUNT - successful_tests, (TEST_COUNT - successful_tests) * 100 / TEST_COUNT);
-	printf("on " COLOR_ARCH "%s" COLOR_DEFAULT "%s\n", sljit_get_platform_name(), sljit_has_cpu_feature(SLJIT_HAS_FPU) ? " (with fpu)" : " (without fpu)");
+
+	fpu = sljit_has_cpu_feature(SLJIT_HAS_FPU);
+	if (simd && fpu)
+		strcpy(features, " (with: fpu, simd)");
+	else if (fpu)
+		strcpy(features, " (with fpu)");
+	else
+		strcpy(features, " (without fpu)");
```

Comment:
this might misrepreset the case where (!fpu && simd), but that is by design as that combination (while possible) is not something that is available in the market yet.

SLJIT_IS_FPU_AVAILABLE and its usecases is also missing and worth discussing, but IMHO might be able to wait until SIMD support is added to sljit proper.